docs/upgrading-solidus/v3.4.mdx

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 1
+---
+
+import PRLink from '@site/src/theme/PRLink';
+import MinimalRequirements from '@site/src/theme/MinimalRequirements';
+
+# Solidus v3.4 (2023-04-TBD)
+
+<MinimalRequirements ruby="2.7" rails="6.0" />
+
+This is a slimmer release, but a very important one as it'll be the last on the 3.x series. [Following our policy](https://solidus.io/release_policy/#last_minor_before_major), there won't be other changes except removing deprecated code before we release v4. That means if you adapt your store to be free of warnings, you should be ready to go when v4 is out (and it's just around the corner!).
```

Comment:
Do you mean `there won't be other changes except removing deprecated code in release v4`? @waiting-for-dev 🤔

Reply:
That's clearer. Updated!